src/Driver/ChromeDriver/Downloader.php

```diff
@@ -35,7 +35,7 @@ final class Downloader implements DownloaderInterface
     private const BINARY_LINUX           = 'chromedriver_linux64';
     private const BINARY_MAC             = 'chromedriver_mac64';
     private const BINARY_WINDOWS         = 'chromedriver_win32';
-    private const DOWNLOAD_ENDPOINT_JSON = 'https://edgedl.me.gvt1.com/edgedl/chrome/chrome-for-testing';
+    private const DOWNLOAD_ENDPOINT_JSON = 'https://storage.googleapis.com/chrome-for-testing-public';
```

Comment:
Did the download URL change for previous versions as well, or are those still on `edgedl.me.gvt1.com`?
We'd want the download for all versions to still work, not just the latest version.

I like your suggestion in #90:
> But possibly we should look into the option to resolve the download url, from the page we use to resolve the versions?

Reply:
The endpoint just returns a 404 now, as it seems to have been taken down completely.

I'll quickly have a look to see if i can find an endpoint from an older version, and see if that returns a 404 as well. 

But https://googlechromelabs.github.io/chrome-for-testing/ just mentions the new URL now. For stable, beta, dev and canary

Reply:
Oof. It seems only the new ones have actually moved. Which means we will need to add an additional check, with a third endpoint :/

Reply:
I'll merge this as a patch version. Would be nice to get a follow-up PR if you're willing to resolve the URL in the same way as the versions.